### PR TITLE
Added SNS and website docs update

### PR DESCRIPTION
**Pull request by kschumy**
*Issue #, if available:*

*Description of changes:*
* Added SNS topic to https://distro.eks.amazonaws.com with the hopes people will subscribe with more visibility
* Fixed minor grammar fixes on the website that have been annoying me for months even though I know they're not a big deal

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/contents/community/faq.md [hunk 1]
```diff
@@ -63,8 +64,8 @@
     What operating system can I install EKS Distro on?
 
 !!! quote "Answer"
-    We provide the same upstream versions of Kubernetes and dependencies which 
-    operating system vendors have tested and confirmed to work with Kubernetes,
+    We provide the same upstream versions of Kubernetes and dependencies that 
+    operating system vendors have tested and confirmed to work with Kubernetes
     and will work the same way. As a result, EKS Distro works with common operating
     systems already used to create Kubernetes clusters, such as CentOS, 
     Canonical Ubuntu, Red Hat Enterprise Linux, Suse, and more. We are testing
```
Comment (kschumy) [inline docs/contents/community/faq.md:71]: Also this is not a valid sentence, but I didn't know what it was trying to say

```
[....] We are testing 
EKS Distro with select vendors to ensure support on Bottlerocket, Amazon Linux 2,
and Canonical has tested EKS Distro for compatibility with Ubuntu.
```

Comment (TerryHowe) [reply docs/contents/community/faq.md:71]: Seems like that should be `EKS Distro is tested by select vendors on various operating systems including Bottlerocket, Amazon Linux 2 and Ubuntu.`

File: docs/contents/community/governance.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 # Project governance
 
 This document lays out the guidelines under which the 
-EKS Distro (EKS-D) project will be governed. 
-The goal is to make sure that the roles and responsibilities
-are well defined and clarify on how decisions are made.
+EKS Distro (EKS-D) project will be governed. The goal 
+is to make sure that the roles and responsibilities
+are well-defined and clarify on how decisions are made.
```
Comment (TerryHowe) [inline docs/contents/community/governance.md:6]:
```suggestion
are well-defined and clarify how decisions are made.
```

File: docs/contents/community/overview.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
 # Overview
 
-We work hard to provide a high-quality Kubernetes distribution just like EKS
-and we greatly value feedback and contributions from our community.  Please
+We work hard to provide a high-quality Kubernetes distribution just like EKS,
```
Comment (TerryHowe) [inline docs/contents/community/overview.md:3]:
```suggestion
We work hard to provide a high-quality Kubernetes distribution for EKS,
```

Maybe?

File: docs/contents/index.md [hunk 4]
```diff
@@ -39,23 +39,24 @@ installing EKS-D.
 The tenets of the EKS Distro (EKS-D) project are:
 
 1. **The Source**: The goal of the EKS Distro is to be the Kubernetes source for EKS and EKS Anywhere
```
Comment (TerryHowe) [inline docs/contents/index.md:41]: Should maybe update the wiki to match?

File: docs/contents/index.md [hunk 5]
```diff
@@ -39,23 +39,24 @@ installing EKS-D.
...
-3. **Opinionated Modularity**: Provide opinionated defaults about the best components to include with Kubernetes, but give customers the ability to swap them out
+2. **Simple**: Make using a Kubernetes distribution simple and boring (reliable and secure)
+3. **Opinionated Modularity**: Provide opinionated defaults about the best components to include with Kubernetes but give customers the ability to swap them out
 4. **Open**: Provide open source tooling backed, validated and maintained by Amazon
-5. **Ubiquitous**: Enable customers and partners to integrate a Kubernetes distribution in the most common tooling (Kubernetes installers and distributions, infrastructure as code, and more).
-6. **Stand Alone**: Provided for use anywhere without AWS dependencies
-7. **Better with AWS**: Enable AWS customers to easily adopt additional AWS services
+5. **Ubiquitous**: Enable customers and partners to integrate a Kubernetes distribution in the most common tooling (Kubernetes installers and distributions, infrastructure as code, and more)
+6. **Stand Alone**: Provide for use anywhere without AWS dependencies
```
Comment (TerryHowe) [inline docs/contents/index.md:46]: Not sure I dig this change

Comment (kschumy) [reply docs/contents/index.md:46]: What don't you like? This is the rationale for each:

* 2 and 5 had periods at the end, but the other bullet points didn't. I removed them to be consistent.
* 3 didn't have an independent clause after the comma, so I removed the comma.
* 6 was past tense, but the rest were present tense. So I changed the tense for consistency.
* 7 had a split infinitive, so I moved the adverb "easily" to the end to fix this.

File: docs/contents/index.md (hunk 4 above)
Comment (kschumy) [reply docs/contents/index.md:41]: Which wiki?

File: docs/contents/index.md (hunk 5 above)
Comment (TerryHowe) [reply docs/contents/index.md:46]: I was just talking about the Provided->Provide change. It isn't really past tense though right? It is just a phrase in the sentence `EKS-D is provided for use anywhere without AWS dependencies`

Comment (TerryHowe) [reply docs/contents/index.md:46]: Maybe it would be more consistent if it just said `for use anywhere without AWS dependencies`

Comment (kschumy) [reply docs/contents/index.md]: I see where you're coming from. I reverted the proposed changes to this in the most recent revision

**Review by TerryHowe — APPROVED** [review]
/lgtm

**Pull request merged**